2021q2/opath.adoc

```diff
@@ -0,0 +1,65 @@
+== VFS path descriptors API
+
+Contact: Konstantin Belousov <kib@FreeBSD.org>
+
+Unix VFS API, from very high point of view, provides two kinds of
+interfaces.  One is the path-based operations, typically manipulating
+metadata, like remove, rename, change permissions and similar.  Second
+is file-descriptor operations, typical are read, write, truncate, and
+other.  The well-known open(2) syscall returns file descriptor from
+path.
+
+The file descriptor is a reliable handle for the file which is
+targeted by operations.  Whatever other action occured to the file
```

Comment:
I'd s/occured to/was performed on/, but even if you don't go with it, s/occured/occurred/ (2 "r"s).